src/backend/commands/functioncmds.c

```diff
@@ -1011,13 +1010,9 @@ interpret_AS_clause(Oid languageOid, const char *languageName,
 	}
 }
 
-
-/*
- * CreateFunction
- *	 Execute a CREATE FUNCTION (or CREATE PROCEDURE) utility statement.
- */
 ObjectAddress
-CreateFunction(ParseState *pstate, CreateFunctionStmt *stmt)
+CreateFunction(ParseState *pstate, CreateFunctionStmt *stmt, bool insideModule,
```

Comment:
Okay, doing that. Originally, I didn't want to create a big diff by refactoring all the common code for the two; and because there is only one call site for CreateFunction.